host_cleanup.sh

```diff
@@ -32,6 +32,14 @@ ansible-playbook \
     -b -vvv ${VM_SETUP_PATH}/teardown-playbook.yml
 
 sudo rm -rf /etc/NetworkManager/dnsmasq.d/openshift-${CLUSTER_NAME}.conf /etc/yum.repos.d/delorean*
+sudo rm -rf /etc/NetworkManager/conf.d/dnsmasq.conf
```

Comment:
A question (@tsorya ?). The code below using the Ansible module `ini_file` seems to indicate the intention to append a new entry in case of existing file; in such case maybe for consistency it could be useful to revert just that change (ie `ansible localhost -b -m ini_file -a "path=/etc/NetworkManager/conf.d/dnsmasq.conf section=main option=dns value=dnsmasq state=absent", rather than removing the entire config file at all?

Reply:
Yeah I thought the same but looking at the file there's no other content so removing seemed OK - also note if you're switching between dev-scripts and metal3-dev-env that it's removed here https://github.com/metal3-io/metal3-dev-env/blob/master/host_cleanup.sh#L56

Reply:
Ok.

/lgtm